.github/workflows/e2e.yml

```diff
@@ -0,0 +1,109 @@
+name: E2E Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  testOnPR:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Copy test environment variables
+        run: |
+          cp example.env .env
```

Comment:
can you add the example.env file

Reply:
Here's the file https://github.com/openmrs/openmrs-esm-form-builder/blob/main/example.env

Reply:
okay